src/common/WixDistribution.cs

```diff
@@ -34,6 +34,11 @@ internal static class WixDistribution
         public static string ShortProduct = "WiX Toolset";
 
         /// <summary>
+        /// Product name for the distribution.
+        /// </summary>
+        public static string Product = "Windows Installer XML Toolset";
```

Comment:
OK I will change it.

On Thu, Jul 28, 2016 at 3:20 AM, Rob Mensching <notifications@github.com>
wrote:

> In src/common/WixDistribution.cs
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72581636>:
>
> > @@ -34,6 +34,11 @@ internal static class WixDistribution
> >          public static string ShortProduct = "WiX Toolset";
> >
> >          /// <summary>
> > +        /// Product name for the distribution.
> > +        /// </summary>
> > +        public static string Product = "Windows Installer XML Toolset";
>
> We're not called "Windows Installer XML Toolset" any more (we should fix
> places that use this old name). It's just "WiX Toolset" now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/5b08dba511d7438508767effab67896442e4ad1b#r72581636>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLUkLxskcR2Tco3jRGZtfIOZUlGoNks5qaGZMgaJpZM4JVm8P>
> .
>